content/en/docs/examples/local-testing-with-kind/index.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Testing Istio Locally With Kubernetes In Docker
+description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation
+weight: 10
+keywords:
+- kubernetes
+- kind
+- metallb
+- cert-manager
+- ingress
+test: true
+---
+
+## About
+
+This example uses Kubernetes In Docker ([KIND][kind]) to deploy Istio locally. The cluster produced is intended to be a supplement for further testing, evaluation and experimentation with Istio.
+
+## Before you begin
+
+Make sure that you have [Docker][docker] and [Kind][kindquickstart] installed and that you are able to create and use kind clusters and connect to them with [Kubectl][kubectl].
+
+## Deploying Kubernetes
+
+For this example we'll be creating a default Kubernetes cluster using [Kind][kind] with a single node:
```

Comment:
Istio documentation avoids "we" terminology. Please change "we" to "you" or reword, here and throughout the doc.
```suggestion
Create a default Kubernetes cluster using [Kind][kind] with a single node:
```

Reply:
Updated all the spots, thank you :)

Reply:
:+1: updated